tests/test_mindspore_classic_solvers.py

```diff
@@ -0,0 +1,325 @@
+# Copyright (c) 2022 Thinklab@SJTU
```

Comment:
Merge this file into "test_classic_solvers.py" and avoid any duplicate functions